server/uitl/validateUserController.js

```diff
@@ -31,6 +31,8 @@ exports.validateUserInfo = async function (req) {
   const availableHoursTo = req.body.availableHoursTo;
   const availableDays = req.body.availableDays;
   const calendarUrl = req.body.calendarUrl;
+  const accessToken = req.body.accessToken;
+  const expiryDate = req.body.expiryDate;
```

Comment:
to make this cleaner, save `req.body` in a variable you can also write this like

`const {calendarUrl, accessToken, expiryDate} = req.body;`